spec/controllers/hyrax/dashboard/works_controller_spec.rb

```diff
@@ -0,0 +1,11 @@
+RSpec.describe Hyrax::Dashboard::WorksController, type: :controller do
+  describe "#search_builder_class" do
+    subject { controller.search_builder_class }
+    it { is_expected.to eq Hyrax::WorksSearchBuilder }
+  end
```

Comment:
no, don't remove the spec.. GH shows you when there is merge conflicts in a PR and won't allow merging if there will be a problem.